CREDENTIAL_CONFIG.yml

```diff
@@ -30,6 +30,10 @@ poc_domain: <CHANGEME>
 htpasswd_username: <CHANGEME>
 htpasswd_password: <CHANGEME>
 
+cert_mail_address: <CHANGEME>
```

Comment:
I think it might be better to rename this var to `certificate_email_address` to be more explicit. There are other vars that contain 'email' rather than 'mail' so would make things more consistent in terms of naming.

Reply:
Or even `cert_email_address`. We seem to use 'cert' rather than 'certificate' in our variable names.